eventuals/http.h

```diff
@@ -547,9 +566,91 @@ struct _HTTP {
                       continuation.closed_ = true;
 
                       if (!continuation.error_) {
+                        // Build headers map.
+                        std::stringstream headers_buffer_stringstream(
+                            continuation.headers_buffer_.Extract());
+                        std::map<std::string, std::string> headers;
+
+                        // Typical 'headers_buffer_stringstream'
+                        // looks like this:
+                        // --------------------------------
+                        // HTTP/1.1 200
+                        // SomeHeaderKey1: SomeHeaderValue1
+                        // SomeHeaderKey2: SomeHeaderValue2
+                        // --------------------------------
+                        while (!headers_buffer_stringstream.eof()) {
+                          std::string line;
+                          std::getline(headers_buffer_stringstream, line);
+
+                          // Find where ':' is.
+                          auto column_pos = std::find(
```

Comment:
What's a `pos`? 🤪 Keep me honest, I'm trying to be use complete words too but sometimes I forget!